services/ui-src/src/libs/actionLib.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Workflow } from "cmscommonlib";
+import RequestTemporaryExtension from "../page/action/RequestTemporaryExtension";
+import RespondToRAI from "../page/action/RespondToRAI";
+import Withdraw from "../page/action/Withdraw";
+
+export const actionComponent = {
```

Comment:
I do like how each action is its own component and that could give us flexibility in the future to handle each differently. In fact i think its brilliant the way you made those components the link portion only such that it can be reused on both package and details view. It might however IMO be even better/cleaner to make actionComponent itself a component rather than an array of components and that parent action component can decide which child component to render based on its inputs. Just a thought.